backend/app/tests/api/routes/documents/test_route_document_remove.py

```
@@ -25,11 +26,11 @@ class TestDocumentRouteRemove:
     @patch("app.api.routes.documents.get_openai_client")
     def test_response_is_success(
         self,
-        mock_get_openai_client,
+        mock_get_openai_client: Any,
```

Ruff (0.14.10)

Undefined name `Any`

(F821)